packages/node-integration-tests/suites/tracing/auto-instrument/mongodb/test.ts

```diff
@@ -3,93 +3,95 @@ import { MongoMemoryServer } from 'mongodb-memory-server';
 
 import { assertSentryTransaction, getEnvelopeRequest, runServer } from '../../../../utils';
 
-let mongoServer: MongoMemoryServer;
+const NODE_VERSION = parseSemver(process.versions.node);
 
-beforeAll(async () => {
-  mongoServer = await MongoMemoryServer.create();
-  process.env.MONGO_URL = mongoServer.getUri();
-}, 30000);
+// Skipping on Node versions below 12 as `mongo-memory-server` supports >= 12
+const conditionalTest = NODE_VERSION.major && NODE_VERSION.major < 12 ? describe.skip : describe;
```

Comment:
Let’s extract this into a helper, turn it into a function, and give the node version as the function argument. That way we can re-use this in other places.